mandatory/4-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(val) {
+  return val+val*0.2;
```

Comment:
please, strive to give variables more self descriptive names. 
`price` is better than `val`;

returned value could be rewritten as : 
```js
return price * 1.2;
```

Reply:
oh, and for better readability, put spaces between variable names and other signs and keywords, please.